wrappers/services.go

```diff
@@ -550,6 +558,12 @@ WantedBy={{.ServicesTarget}}
 		restartCond = snap.RestartOnFailure.String()
 	}
 
+	const baseScore = 899
```

Comment:
could you add a comment why we don't let this start at 1001, and thus get a score of `-(1001-1) = -1000` which means never kill?

also why start at -898 and not -900 ?

Reply:
I added a comment about this now: 
	// start with score -899 because snapd itself has OOMScoreAdjust=-900

Reply:
but effective vitalityRank starts at 1 so: 900-1 = 899 ,  now 899-1 = 898, or am I confused?